src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -27,6 +29,10 @@ const amountToLendQuery = gql`
 		my {
 			id
 			depositIncentiveAmountToLend
+			userAccount {
+				id
+				hasDepositIncentive
```

Comment:
It looks like `hasDepositIncentive` hasn't been implemented yet in the BE, so I'll hold off on approving until the BE is ready and deployed to dev.